tobler/dasymetric/raster_tools.py

```diff
@@ -10,12 +10,15 @@
 import rasterio as rio
 import rasterstats as rs
 from joblib import Parallel, delayed
+from packaging.version import Version
 from rasterio import features
 from rasterio.mask import mask
 from shapely.geometry import shape
 
 from ..util.util import _check_presence_of_crs
 
+GPD_10 = Version(gpd.__version__) >= Version("0.99.0")
```

Comment:
is `0.99.0` simply for excluding future patch releases of `>=1.0`? If so, perhaps `0.99.99`?

Reply:
If I do 1.0.0, the current alpha is not considered equal, so this is one way of saying 1.0 or newer, including dev and pre-release versions. But I can also change that to something more explicit. I don't expect patch releases on 0.14.x branch will happen anyway.